packages/web/src/scss/components/Modal/_theme.scss

```
@@ -16,5 +16,7 @@ $content-background-color: tokens.$background-basic;
 $content-shadow: tokens.$shadow-300;
 
 $header-spacing: tokens.$space-600;
+$footer-spacing: tokens.$space-600;
+$gap-spacing: tokens.$space-400;
```

```suggestion
$footer-gap: tokens.$space-400;
```
IMHO this might be better